.github/workflows/linux-browser-compatibility-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Linux Browser Compatibility Test
+
+on:
+  pull_request:
+      branches:
+        - main
+        - release-1.x
+
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
+  TEST_TYPE: Browser-Compatibility-Test
```

Comment:
Do we need to change this to `Linux-Browser-Compatibility-Test` to differentiate it from the desktop Browser Compatibility Test?

Reply:
They will be running from the same `yml` file so we cant change it. I have kept Linux tests as separate jobs to make it easier to debug.